framework/src/processing/lib/spark-runtime/emr-containers/eks-support-controllers-version.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import { KubernetesVersion } from 'aws-cdk-lib/aws-eks';
+
+//The version mapping can be taken from here
```

Comment:
this kind of comment is not taken into account for the doc, use the syntax:
```
/**
 *
 **/
```

Reply:
Will fix the comment format, this is supposed to be internal only, the links are for maintainers in the future to know where to get the versions.

Reply:
then no need, you can let them like this